superset/cli.py

```diff
@@ -31,6 +31,9 @@ def init():
     '-d', '--debug', action='store_true',
     help="Start the web server in debug mode")
 @manager.option(
```

Comment:
It's not well documented but it's similar to argparse, so you can ` action='store_true'` as in
```
@manager.option(
    '-n', '--no-reload', action='store_true', dest='no_reload',
    help="Don't use the reloader in debug mode")
```